rtbkit/plugins/augmentor_interface/http_augmentor_interface.cc

```diff
@@ -0,0 +1,486 @@
+#include "http_augmentor_interface.h"
+#include "jml/db/persistent.h"
+#include "soa/service/http_client.h"
+#include "soa/utils/generic_utils.h"
+#include "rtbkit/common/messages.h"
+#include "rtbkit/plugins/bid_request/openrtb_bid_request_parser.h"
+#include "rtbkit/openrtb/openrtb_parsing.h"
+#include "rtbkit/core/router/router.h"
+#include "jml/arch/futex.h"
+
+using namespace Datacratic;
+using namespace RTBKIT;
+
+namespace {
+    DefaultDescription<OpenRTB::BidRequest> desc;
+
+    std::string httpErrorString(HttpClientError code)  {
+        switch (code) {
+            #define CASE(code) \
+                case code: \
+                    return #code;
+            CASE(HttpClientError::None)
+            CASE(HttpClientError::Unknown)
+            CASE(HttpClientError::Timeout)
+            CASE(HttpClientError::HostNotFound)
+            CASE(HttpClientError::CouldNotConnect)
+            CASE(HttpClientError::SendError)
+            CASE(HttpClientError::RecvError)
+            #undef CASE
+        }
+        ExcCheck(false, "Invalid code path");
+        return "";
+    }
+}
+
+namespace RTBKIT {
+
+Logging::Category HttpAugmentorInterface::print("HttpAugmentorInterface");
+Logging::Category HttpAugmentorInterface::error(
+    "HttpAugmentorInterface Error", HttpAugmentorInterface::print);
+Logging::Category HttpAugmentorInterface::trace(
+    "HttpAugmentorInterface Trace", HttpAugmentorInterface::print);
+
+}
+
+auto HttpAugmentorInterface::readFormat(const std::string& fmt) -> Format {
```

Comment:
You can remove it, it should not be needed, custom formats are getting removed from rtbkit and will be kept private to Datacratic :)